kernel/Utilities/GlobalIndexing.h

```diff
@@ -44,6 +44,9 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "../Base/Face.h"
 #include "../Base/MeshManipulatorBase.h"
 #include "../Base/Node.h"
+
+#include "Base/MeshEntity.h"
```

Comment:
weird why is the path different

Reply:
The others are historically weird, this is the more correct one. But I'll fix them all.